libsol/include/sol/transaction_summary.h

```diff
@@ -69,6 +72,9 @@ SummaryItem* transaction_summary_nonce_account_item();
 SummaryItem* transaction_summary_nonce_authority_item();
 SummaryItem* transaction_summary_general_item();
 
+
+
+
```

Comment:
remove whitespace change

Reply:
Thanks for the comment, I think that one commit with formatting fixes was lost somewhere when i was cherry picking. I'll handle those cases